recipe/meta.yaml

```
@@ -21,20 +21,18 @@ source:
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  # NOTE-TO-SELF: maybe I don't need to skip these anymore
   skip: True  # [py<37]
   skip: True  # [linux and (s390x or ppc64le)]
 
 requirements:
   host:
-    - python
+    - python >=3.6
```

This is already taken care of by `skip: True  # [py<37]`
```suggestion
    - python
```

---

will do